includes/class-newspack-ads.php

```diff
@@ -73,6 +73,9 @@ private function includes() {
 		include_once NEWSPACK_ADS_ABSPATH . '/includes/class-newspack-ads-custom-label.php';
 		include_once NEWSPACK_ADS_ABSPATH . '/includes/class-newspack-ads-bidding.php';
```

Comment:
Suggestion: move the `defined( 'NEWSPACK_ADS_EXPERIMENTAL_BIDDERS' )` check here, instead of repeating it in all four bidder files.